src/LogOutputManager.ts

```diff
@@ -258,13 +258,18 @@ export class LogOutputManager {
             this.displayedLogLines.push(logLine);
             let match = this.pkgRegex.exec(logLine.text);
             if (match) {
-                const pkgPath = match[1];
-                const lineNumber = Number(match[2]);
-                const filename = this.getFilename(pkgPath);
-                const extension = pkgPath.substring(pkgPath.length - 4);
-                let customText = this.getCustomLogText(pkgPath, filename, extension, Number(lineNumber), logLineNumber);
-                const customLink = new CustomDocumentLink(logLineNumber, match.index, customText.length, pkgPath, lineNumber, filename);
-                this.docLinkProvider.addCustomLink(customLink);
+                const isFilePath = match[2] === 'file://';
```

Comment:
the file protocol is more complicated than this. Sometimes it can be `file:/`, `file://`, `file:///`, `file:////`. (See [how many slashes on wikipedia](https://en.wikipedia.org/wiki/File_URI_scheme#How_many_slashes?).  We should be using a tool to handle this. [file-url](https://github.com/sindresorhus/file-url) handles parsing things into file urls, there's also the vscode Uri class that takes a uri and provides the various parts (like scheme, path ,etc...).  

at the very least, perhaps consider a regular expression match on `match[2]` instead. 
```
match[2].match(
    /file:\/{1,4}/
)
```

Reply:
this is a moot point. note the origina regex; it only catches 2 slashes, which are matched here the rest is the path.
that will always be valid, even on windows, as per the wiki link you shared states: "Note that the drive letter followed by a colon and slash is part of the acceptable file URI."

so we don't need to do anything here; did you experience an issue with this?